### DISCUSSION TITLE
SEB version 3.6.0

### DISCUSSION THREAD

**Discussion by IwisGiwis** [SafeExamBrowser/seb-win-refactoring#633] [General]
Dear SEB Team,

We (University of Innsbruck) are planning to update the SEB version on our systems for the next semester starting in September. In the last years you released an update in August. Are you also planning to release the 3.6.0 version in August? Is there also a beta version that we can try out?

Best regards,
Stefan

**Comment by dbuechel** [comment]
We do at the moment not (yet) have a fixed timeline for the release of SEB 3.6.0, thus it's quite unlikely (albeit not impossible) that we'll make the release until the end of August. But there indeed is a beta version which can be tested: https://sebdev-let.ethz.ch/project/appveyor/seb-win-refactoring (Platform x86 > Artifacts > [...]SetupBundle.exe). Please note that a beta version should under no circumstances be used in a productive environment.

**Reply by IwisGiwis** [reply]
Thanks, I must have been blind for not finding the Artifacts submenu. Using the signature for permitted processes seems to work fine. Is there a possibility that it will be also implemented for prohibited processes?

**Reply by dbuechel** [reply]
> Using the signature for permitted processes seems to work fine.

Excellent, that's great to hear.

> Is there a possibility that it will be also implemented for prohibited processes?

At the moment, we do not have any plans to do so, as any not explicitly allowed application will already either be hidden or terminated by SEB.

**Reply by IwisGiwis** [reply]
I would have hoped you could also block the signatures of programs without original filenames like AnyDesk, which can run in the background and control the device from a remote user. I don't know if this is possible or could be bypassed but I think it would be an improvement even if signatures change after some time.

**Reply by dbuechel** [reply]
We were thinking about that scenario, but the problem is that unlike with permitted applications, only a signature by itself is not enough information to meaningfully block resp. determine a prohibited application.

As an example, say you'd like to block MSFT Edge, but a student renames the executable to something else. Now the only thing you'd have left would be the signature, but as that likely is some MSFT signature used for many (maybe even system) applications, then SEB would try to terminate a whole host of different (system) applications which would most definitely cause severe issues or even make the system unstable or crash.

That's why we don't have any plans to incorporate the signature verification into the monitoring of prohibited applications, as these as already stated will anyway either be hidden or terminated by SEB.

**Comment by 1TheITCrowd** [comment]
Hello,

Is there a timeline on version 3.6?
Just asking because students are using a cheat for version 3.5 and exams are up in a month or 2.

Best regards!

**Reply by dbuechel** [reply]
We were aiming for end of October / beginning of November, but due to various unforeseen issues the release will be delayed for at least a month if not longer.
